modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx

```diff
@@ -136,21 +136,27 @@ export default function EditNotificationTemplate({
 			!Liferay.FeatureFlags['LPD-11165'] &&
 			notification.type === 'email'
 		) {
-			const recipients = notification.recipients[0] as EmailRecipients;
+			const recipient = notification.recipients[0] as EmailRecipients;
+
+			let newRecipient = {
+				bcc: recipient.bcc,
+				cc: recipient.cc,
+				from: recipient.from,
+				fromName: recipient.fromName,
+				singleRecipient: recipient.singleRecipient,
+				to: recipient.to,
+			} as EmailRecipients;
+
+			if (Liferay.FeatureFlags['LPD-21580']) {
+				newRecipient = {
+					...recipient,
+					usersLanguage: recipient.usersLanguage,
+				};
+			}
```

Comment:
here you are going to lose everything you put inside `newRecipient` when you do the `...recipient`, is this really expected?

Reply:
Hey @rebsilva,
good catch, this is not the expected I changed it.